examples/globe.js

```diff
@@ -55,7 +55,166 @@ if (!renderer) {
 // Add one imagery layer to the scene
 // This layer is defined in a json file but it could be defined as a plain js
 // object. See Layer* for more info.
-promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb));
+var orthoLayerOptions = {
+    // Choose here a unique id for your layer
+    "id":         "Ortho",
+    // we'll display this layer as color texture on our globe
+    "type": "color",
+    // protocol and url defines the datasource
+    "protocol":   "wmts",
+    // The api key 'va5orxd0pgzvq3jxutqfuy0b' is for testing purpose, and only usable in localhost.
+    // When you'll host your application based on iTowns, you'll have to request a new api key if you want to use IGN datasources.
+    "url":        "http://wxs.ign.fr/va5orxd0pgzvq3jxutqfuy0b/geoportail/wmts",
+    // this is the name of the layer to use in the above endpoint
+    // (cf http://wxs.ign.fr/va5orxd0pgzvq3jxutqfuy0b/geoportail/wmts?service=wmts&version=1.0.0&request=GetCapabilities)
+    "name": "ORTHOIMAGERY.ORTHOPHOTOS",
+    // this option will be passed to the loaders that'll do the individual fetching
+    // see Fetcher.js
+    "networkOptions": {
+        "crossOrigin": "anonymous"
+    },
+    //
+    "updateStrategy": {
+        "type": itowns.STRATEGY_MIN_NETWORK_TRAFFIC,
+        "options": {}
+    },
+    // what format to use, among the supported format of this layer
+    "format": "image/jpeg",
+
+    "options": {
+        // store there your attributions for your datasource
+        "attribution" : {
+            "name":"IGN",
+            "url":"http://www.ign.fr/"
+        },
+        // This defines the tilematrixset of our WMTS layer. Supported types are currently PM and WGS84G.
+        // More to be supported in the futur.
+        "tileMatrixSet": "PM",
+    }
+};
+
+// now we need to set the WMTS limits. This should be compatible from what GetCapabilities returns.
+// (cf http://wxs.ign.fr/va5orxd0pgzvq3jxutqfuy0b/geoportail/wmts?service=wmts&version=1.0.0&request=GetCapabilities)
+orthoLayerOptions.options.tileMatrixSetLimits = {
+    "2": {
```

Comment:
For this verbose definition, we can add a auto-hide functionality to the generated webpage. It could only display like 10 lines by default, then the user can unfold it if he wants to see everything.

Reply:
You could also let the first one expanded, and write in a more compact version the others:
`"3": { "minTileRow": 0, "maxTileRow": 8, "minTileCol": 0, "maxTileCol": 8 }`